### PR TITLE
Refactors so TableInvoice takes the total fee

### PR DESCRIPTION
**Pull request by thebkr7**
Closes #28 for the Extension Repo

**Description:**

Refactors so TableInvoice takes the total fee rather than taking two numbers as props and calculating it there.

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by thebkr7** [comment]
Will change destination branch to develop once the chrome-extension branch has been merged.

**Comment by thebkr7** [comment]
Refactoring TableInvoice accepting BondDenom as a prop based on comment: https://github.com/luniehq/lunie-browser-extension/issues/30#issuecomment-507854375

**Comment by thebkr7** [comment]
The above refactor of TableInvoice was completed in PR https://github.com/luniehq/lunie/pull/2802

File: test/unit/specs/components/ActionModal/components/__snapshots__/TableInvoice.spec.js.snap [hunk 1]
```diff
@@ -22,7 +25,7 @@ exports[`TableInvoice should show the transaction invoice summary 1`] = `
        
       <span>
         
-        0.030864
+        NaN
```
Comment (jbibla) [inline test/unit/specs/components/ActionModal/components/__snapshots__/TableInvoice.spec.js.snap]: this looks incorrect

Comment (thebkr7) [reply test/unit/specs/components/ActionModal/components/__snapshots__/TableInvoice.spec.js.snap]: Thanks. The test wasn't updated to account for the new props.

File: test/unit/specs/components/ActionModal/components/__snapshots__/ActionModal.spec.js.snap [hunk 2]
```diff
@@ -98,8 +98,7 @@ exports[`ActionModal should show the action modal when user has logged in with e
      
     <tableinvoice-stub
       amount="0"
-      gasestimate="0"
-      gasprice="2.5e-8"
+      estimatedfee="0"
```
Comment (jbibla) [inline test/unit/specs/components/ActionModal/components/__snapshots__/ActionModal.spec.js.snap]: should this be 0?

Comment (faboweb) [reply test/unit/specs/components/ActionModal/components/__snapshots__/ActionModal.spec.js.snap]: when the estimate was 0 before then yes ;)

**Pull request merged**